pyspectrafuse/common/msp_utils.py

```
@@ -74,15 +74,17 @@ def get_msp_fmt(row: pd.Series) -> str:
         name_val = row['peptidoform']
         mw_val = row['pepmass']
-        num_peaks_val = len(ast.literal_eval(row['mz_array']))
+        mz_arr = row['mz_array']
+        if isinstance(mz_arr, str):
+            mz_arr = ast.literal_eval(mz_arr)
+        num_peaks_val = len(mz_arr)
         comment_val = f'clusterID={MspUtil().usi_to_uuid(row["usi"])} Nreps={row["Nreps"]} PEP={row["posterior_error_probability"]}'
-        mz_intensity_val = Parquet2Mgf.get_mz_intensity_str(row['mz_array'], row['intensity_array'])
+        mz_intensity_val = '\n'.join(
+            f'{mz} {intensity}'
+            for mz, intensity in zip(row['mz_array'], row['intensity_array'])
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)